oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -299,11 +300,11 @@ private String getProcessorID(String stepping, String model, String family, Stri
             if (checkLine.contains(marker) && checkLine.trim().startsWith("0x00000001")) {
                 String eax = "";
                 String edx = "";
-                for (String register : checkLine.split("\\s+")) {
+                for (String register : ParseUtil.whitespace.split(checkLine)) {
                     if (register.startsWith("eax=")) {
-                        eax = register.replace("eax=0x", "");
+                        eax = StringUtils.replace(register, "eax=0x", "");

```

Comment:
Just as a small help, the following is the code from the Apache Commons StringUtils.replace(String, String, String) method simplified, to avoid performance hits from the varied uses available, to just replacing all the occurences of the search string in the original value:

`/**
* <p>Replaces all occurrences of a String within another String.</p>
*
* <p>A {@code null} reference passed to this method is a no-op.</p>
*
* <pre>
* StringUtils.replace(null, *, *)        = null
* StringUtils.replace("", *, *)          = ""
* StringUtils.replace("any", null, *)    = "any"
* StringUtils.replace("any", *, null)    = "any"
* StringUtils.replace("any", "", *)      = "any"
* StringUtils.replace("aba", "a", null)  = "aba"
* StringUtils.replace("aba", "a", "")    = "b"
* StringUtils.replace("aba", "a", "z")   = "zbz"
* </pre>
*
* @param text  text to search and replace in, may be null
* @param searchString  the String to search for, may be null
* @param replacement  the String to replace it with, may be null
* @return the text with any replacements processed,
*  {@code null} if null String input
*/
private static String replace(final String text, String searchString, final String replacement, int max, final boolean ignoreCase) {
    if (text == null || text.isEmpty() || searchString == null || searchString.isEmpty() || replacement == null) {
        return text;
    }

    int start = 0;
    int end = text.indexOf(searchString, start);

    if (end == -1) {
        return text;
    }

    final int replLength = searchString.length();
    int increase = replacement.length() - replLength;
    increase = increase < 0 ? 0 : (increase * 16);

    final StringBuilder buf = new StringBuilder(text.length() + increase);
    while (end != -1) {
        buf.append(text.substring(start, end)).append(replacement);
        start = end + replLength;
        end = text.indexOf(searchString, start);
    }
    buf.append(text.substring(start));
    return buf.toString();
}`

The only thing missing would be any needed reference to the code being leveraged from the Apache Commons v3.6